client/web/compose/src/components/PageBlocks/Navigation/NavTypes/Url.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <tr>
+    <td />
+    <td>
+      <b-form-group :label="$t('navigation.fieldLabel')">
+        <b-form-input
+          v-model="value.options.itemOption.label"
+          type="text"
+          size="sm"
+        />
+      </b-form-group>
+    </td>
+    <td>
+      <b-form-group :label="$t('navigation.url')">
+        <b-form-input
+          v-model="value.options.itemOption.url"
+          type="text"
+          size="sm"
+        />
+      </b-form-group>
+    </td>
+    <td>
+      <b-form-group :label="$t('navigation.openIn')">
+        <b-form-select
+          v-model="value.options.itemOption.targetType"
```

Comment:
i think we can call the itemOption just item. And then hold all options inside it, same for dropdown